### ISSUE TITLE
Incorrect Table Height when Displaying in REPL (w/ Blink)

### ISSUE THREAD

**Issue by jacobadenbaum** [JuliaComputing/TableView.jl#14]
When displaying the table in the REPL using blink (as suggested in the README), the table doesn't scale to fit the window, which can be annoying if you want to resize it/move it around, etc...

It looks like this is because the default behavior is to set the height to a [fixed number](https://github.com/JuliaComputing/TableView.jl/blob/e528dfc18b2160b6f95306518e99d2462b4fc6a8/src/TableView.jl#L33).  I think this can be fixed by setting `height = "100vh"` (see [this](https://stackoverflow.com/a/28121246) answer on stack overflow).  

Should this (or something like it) be the default?

**Comment by pfitzseb** [comment]
No, this unfortunately can't be the default behaviour because it breaks in other contexts (e.g. Jupyter).

I guess we could try to make the defaults context sensitive (i.e. try to figure out where we're being run and set the defaults accordingly), but imho that's the wrong solution.
There probably is a clean solution to this, but a) not with the current WebIO and b) I can't figure it out.

**Comment by jacobadenbaum** [comment]
Yeah, I figured it must be something like that. I would up wrapping the correct solution that makes a blink window and populates it output that has autoscaling height in a little 10 line [package](https://github.com/jacobadenbaum/TableBrowse.jl) in case anyone else is annoyed by this.
